src/main/java/com/pscheduler/server/model/GpaCourse.java

```diff
@@ -0,0 +1,42 @@
+package com.pscheduler.server.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Entity
+public class GpaCourse {
```

Comment:
Let's have the same private variables as the columns in the grade distribution:

![image](https://user-images.githubusercontent.com/17682577/60392833-9206a300-9ad9-11e9-95b7-7cc1390b46c1.png)

Match names with the other course model. Let's use "students", "A", "B", etc

Reply:
Need you to explain the 2nd line some more.

Reply:
Done